packages/sdk-react/src/core/hooks/useAutosizeGridColumns.ts

```diff
@@ -10,22 +10,41 @@ import { useGridApiRef } from '@mui/x-data-grid';
 // DOM access is required to accurately calculate dimensions
 export function useAutosizeGridColumns(rows: any) {
   const gridApiRef = useGridApiRef();
+  const [columnWidths, setColumnWidths] = useState<
+    { field: string; width: number }[]
+  >([]);
+
   useEffect(() => {
-    setTimeout(() => {
-      ReactDOM.flushSync(() => {
-        setTimeout(() => {
-          if (gridApiRef.current?.autosizeColumns) {
-            // noinspection JSIgnoredPromiseFromCall
-            gridApiRef.current?.autosizeColumns({
-              columns: ['amount'],
+    const grid = gridApiRef.current;
+    if (!grid || !grid.autosizeColumns) return;
+    if (columnWidths && columnWidths.length > 0) {
+      for (const column of columnWidths) {
+        grid.setColumnWidth(column.field, column.width ?? 100);
+      }
+    } else {
+      setTimeout(() => {
+        ReactDOM.flushSync(() => {
```

Comment:
I recommend changing `*` imports to named imports:
```ts
import { flushSync } from 'react-dom';
```

Reply:
don